continuous_integration/appveyor/environment.yaml

```diff
@@ -1,10 +1,11 @@
 name: testenv
 channels:
   - conda-forge
+  - defaults
 dependencies:
   # required dependencies
   - python=3.6.*
-  - numpy>=1.13.0
+  - numpy<=1.16.4
```

Comment:
```suggestion
  - numpy<=1.16.4    # >= 1.17.0 breaks from https://github.com/pydata/sparse/issues/257
```

Reply:
Would tweak this slightly to keep the lower bound (please see below). Unfortunately wasn't able to add that suggestion to this thread.